### PR TITLE
Add detector implementation

### PR DESCRIPTION
**Pull request by stefanDeveloper**
Solves #14 and #12 

Currently, this is a first draft to init the basic implementation. Later, when our models are trained, adaptions have to be made.

### CODE REVIEW

File: src/detector/detector.py [hunk 1]
```diff
@@ -1,3 +1,206 @@
...
+    """
+    Creates the :class:`Detector` instance. Starts a loop that continously fetches data.
+
+    Args:
+        one_iteration (bool): For testing purposes: stops loop after one iteration
+
+    Raises:
+        KeyboardInterrupt: Execution interrupted by user. Closes down the :class:`LogCollector` instance.
+    """
+    logger.info("Starting Inspector...")
```
Comment (lamr02n) [inline src/detector/detector.py:173]: I suppose that should say ``Starting Detector...``

File: src/detector/detector.py [hunk 2]
```diff
@@ -1,3 +1,206 @@
...
+    detector = Detector()
+    logger.info(f"Inspector is running.")
```
Comment (lamr02n) [inline src/detector/detector.py:175]: Same as above (``Detector is running.``, ``f`` can also be removed)

**Review by lamr02n — APPROVED** [review]
Just two little changes in the logging messages, other than that it looks great.

File: src/detector/detector.py (hunk 2 above)
Comment (stefanDeveloper) [reply src/detector/detector.py:175]: Crtl + C, Crtl + V error :smile:

**Comment by stefanDeveloper** [comment]
> Just two little changes in the logging messages, other than that it looks great.

Fixed!

**Pull request merged**